File: problemset/Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
      int idx = 0;
      for (int i = 0; i < nums.size(); ++i) {
        if (!i || nums[i] != nums[i-1])
          nums[idx++] = nums[i];
      }
      return idx;
    }
};

File: activity/20200805/scau-flyingbird/RemoveDuplicatesFromSortedArray.go
func removeDuplicates(nums []int) int {
	i, j, l := 0, 1, len(nums)
	if l < 2 {
		return l
	} else {
		for j < l {
			if nums[i] != nums[j] {
				i++
				nums[i] = nums[j]
			}
			j++
		}
		return i + 1
	}
}
